docs/spec/channel/version_070.md

```diff
@@ -0,0 +1,140 @@
+
+```
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+ name: kafkachannels.messaging.knative.dev
+ labels:
+    knative.dev/crd-install: "true"
+    messaging.knative.dev/subscribable: "true"
+spec:
+  group: messaging.knative.dev
+  version: v1alpha1
+  names:
+    kind: KafkaChannel
+    plural: kafkachannels
+    singular: kafkachannel
+    categories:
+    - all
+    - knative
+    - messaging
+    - channel
+    shortNames:
+    - kc
+  scope: Namespaced
+...
+```
+
+Each channel is _namespaced_ and MUST have the following:
+
+* label of `messaging.knative.dev/subscribable: "true"`
+* a `channel` category
+
+#### Broker and Triggers
+
+TODO
+
+### Data Plane
+
+The data plane describes the input and output flow of a _Channel_. All Channels exclusively communicate using CloudEvents.
+
+#### Input
+
+Every Channel must expose either an HTTP or HTTPS endpoint. It MAY expose both. The endpoint(s) MUST conform to [HTTP Transport Binding for CloudEvents - Version 0.2](https://github.com/cloudevents/spec/blob/v0.2/http-transport-binding.md). It MUST support both Binary Content mode and Structured Content mode. The HTTP(S) endpoint MAY be on any port, not just the standard 80 and 443. Channels MAY expose other, non-HTTP endpoints in addition to HTTP at their discretion (e.g. expose a gRPC endpoint to accept events).
+
+##### Generic
+
+If a Channel receives an event queueing request and is unable to parse a valid CloudEvent, then it MUST reject the request.
+
+Every event queueing request to the Channel will come with a bearer token, likely a JWT. The bearer token MUST be validated before any other work is done on the request. The specifics of how and what to validate will be identical to Broker ingress verification, which is being [defined](https://github.com/knative/eventing/issues/705#issuecomment-496722527).
+
+The Channel MUST pass through all tracing information as CloudEvents attributes. In particular, it MUST translate any incoming OpenTracing or B3 headers to the [Distributed Tracing Extension](https://github.com/cloudevents/spec/blob/v0.2/extensions/distributed-tracing.md). The Channel SHOULD sample and write traces to the location specified in [`config-tracing`](https://github.com/cloudevents/spec/blob/v0.2/extensions/distributed-tracing.md).
+
+##### HTTP
+
+Channels MUST reject all HTTP event queueing requests with a method other than POST responding with HTTP status code `405 Method Not Supported`. Non-event queueing requests (e.g. health checks) are not constrained.
+
+The HTTP event queueing request's URL MUST correspond to a single, unique Channel at any given moment in time. This MAY be done via the host, path, query string, or any combination of these. This mapping is handled exclusively by the Channel implementation, exposed via the Channel's `status.address`. If an HTTP event queueing request's URL does not correspond to an existing Channel, then the Channel MUST respond with 404 Not Found.
```

Comment:
```suggestion
The HTTP event queueing request's URL MUST correspond to a single, unique Channel at any given moment in time. This MAY be done via the host, path, query string, or any combination of these. This mapping is handled exclusively by the Channel implementation, exposed via the Channel's `status.address`. If an HTTP event queueing request's URL does not correspond to an existing Channel, then the Channel MUST respond with `404 Not Found`.
```